c10/core/DispatchKey.h

```diff
@@ -47,6 +47,9 @@ enum class DispatchKey : uint8_t {
   SparseCPUTensorId, // PyTorch only
   SparseCUDATensorId, // PyTorch only
 
+  // Custom pseudorandom number generator
```

Comment:
This comment could definitely be beefed up a little more. Also, is this a PrivateUse dispatch key, or are we allocating it specifically for secure PRNG? It's good to be clear about this, since you will be an unhappy camper if other people start using this key improperly.

Reply:
This comment can still be beefed up...